src/cr/cube/dimension.py

```diff
@@ -57,6 +57,30 @@ def apparent_dimensions(self):
         """
         return _ApparentDimensions(all_dimensions=self._dimensions)
 
+    @lazyproperty
+    def dimension_order(self):
+        """Tuple of int representing of the dimension order.
+
+        The dimension order depends on the presence of numeric array in the dimensions
+        and the number of the cube dimensions. In case of 3 dimensions e.g.
+        NUM_ARR_X_MR_SUBVAR_X_MR_CAT the order should be (1,2,0) that is basically
+        swapping the MR (2 dimensions) with the NUM_ARRAY dimension. In case of 2
+        dimensions the dimension order correspond simpy to the reverse of the original
+        dimension order.
+        """
+        # NOTE: this is a temporary hack that goes away when we introduce the dim_order
+        # concept. We should receive the actual order directly in the cube_response.
+        # So, all this logic will be deleted.
+        dimension_types = tuple(d.dimension_type for d in self._dimensions)
+        dim_order = tuple(idx for idx in range(len(self._dimensions)))
```

Comment:
can just be `tuple(range(len(self._dimensions)))`

Reply:
yes ... sorry i changed it but not pushed ... damn ... DONE now